src/test/java/org/junitpioneer/internal/TestParameterContext.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.internal;
+
+import static java.util.Objects.requireNonNull;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Parameter;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.platform.commons.support.AnnotationSupport;
+
+public class TestParameterContext implements ParameterContext {
```

Comment:
Where is this used? Why is this here? I've deleted it on my local copy and the build is successful, so I think this should be deleted.